--- linux-3.10.49/drivers/net/usb/qmi_wwan.c	2015-03-13 13:40:17.157580613 +0000
+++ linux-3.10.49/drivers/net/usb/qmi_wwan.c	2015-03-13 16:31:46.537086431 +0000
@@ -753,6 +753,7 @@
 	{QMI_FIXED_INTF(0x413c, 0x81a4, 8)},	/* Dell Wireless 5570e HSPA+ (42Mbps) Mobile Broadband Card */
 	{QMI_FIXED_INTF(0x413c, 0x81a8, 8)},	/* Dell Wireless 5808 Gobi(TM) 4G LTE Mobile Broadband Card */
 	{QMI_FIXED_INTF(0x413c, 0x81a9, 8)},	/* Dell Wireless 5808e Gobi(TM) 4G LTE Mobile Broadband Card */
+	{QMI_FIXED_INTF(0x1e0e, 0x9001, 5)},	/* SIMCom 72X0 */
 
 	/* 4. Gobi 1000 devices */
 	{QMI_GOBI1K_DEVICE(0x05c6, 0x9212)},	/* Acer Gobi Modem Device */
